content/Dev/remote_settings_history.rst

```diff
@@ -0,0 +1,179 @@
+
+A dumb pipe of static data, with complex workflows outside the system itself. A little bit `like this blog </publish-your-pelican-blog-on-github-pages-via-travis-ci.html>`_ actually.
+
+Why not!
+
+
+Thoughts
+========
+
+I hope this long article helped you understand how a long term project can evolve and mutate. I thought it would be interesting to see it from within a company like Mozilla, often mentioned in headlines.
+
+
+What made the success?
+----------------------
+
+*Remote Settings* became a critical part of Firefox. Clearly, since it is leveraging only a subset of *Kinto*, the success can be largely attributed to the vision and efforts of our early team. We're still friends and I'm super proud of what we accomplished! Big up!
+
+The patience of stakeholders who hold our hand in order to land patches in Firefox massively contributed to the success. Special thanks to Gijs, Lina, Florian, mgoodwin, standard8...
+
+Another factor is that we were consistently protecting the functional scope of the system. It is a data pipe: publish data on one side, reach the target audience reliably. It took a lot of effort to say «no», and keep the complexity trend downwards, instead of building dedicated features for specific use-cases.
+
+During our company meetings, we were making sure we would spend more time with people, and the least in front of computers. I don't cross the world to do what I can do from home! We had joyful moments and this happiness certainly contributed to the success :) Through the numerous use-cases, I got to know many teams in the company, and always worked in a fantastic atmosphere. I am happy to help, they seem happy with the solution, it is great!
+
+
+What about Kinto?
+-----------------
+
+I don't want this article or section to sound like a retrospective of *Kinto* itself, but I think there are a few important things to underline.
+
+The `Kinto organization <https://github.com/Kinto/>`_ is relatively quiet, but we have users, and `Dylan <https://dstaley.com/>`_ is dedicating an amazing amount of energy on the project! A lot of engineers at Mozilla are going to benefit from his efforts to migrate the Admin UI to Bootstrap 4 (3 years after the PR was started 🤣)! Kudos!
+
+Nevertheless, we cannot deny that the *Kinto* community is not as flourishing as it used to be ;) And let's be clear: from the Mozilla standpoint, *Kinto* is just an implementation detail.
+
+If I win the lottery and leave (the positive version of the bus factor idea), there is no guarantee that the next pair of eyes taking a fresh look at the *Remote Settings* architecture or the *Kinto* code base will decide to keep it. In other words, as long as *Kinto* is used in *Remote Settings*, the project will be maintained and be taken care of. The day something else is used, Mozilla won't affect any resources. Undoubtedly.
```

Comment:
I think you're using `affect` in the French sense here, so maybe `assign` instead? But even that doesn't sound right since actually Mozilla would take away the resources it's spending on Kinto already, right? Maybe `Mozilla will stop supporting Kinto`.